src/windows.rs

```diff
@@ -339,7 +339,7 @@ ffi! {
         pub fn GetCurrentProcess() -> HANDLE;
         pub fn GetCurrentThread() -> HANDLE;
         pub fn RtlCaptureContext(ContextRecord: PCONTEXT) -> ();
-        pub fn LoadLibraryA(a: *const i8) -> HMODULE;
```

Comment:
Is the A version not supported on UWP?

Reply:
It isn't, but I just realized that I was bitten by `libwinstorecompat` once again. LoadLibraryW isn't available either, and `LoadPackagedLibrary` needs to be used instead. In any case, that function also takes wide strings though

Reply:
But that means that dbghelper must be packaged in the app... which is probably not an option.

I'm afraid the only way to have this built on UWP is to disable it :/